python/bindings/bindings.cpp

```diff
@@ -43,4 +48,5 @@ NB_MODULE(bindings, m)
     init_novatel_oem_messages(messages_mod);
     nb::module_ enums_mod = m.def_submodule("enums", "Enumerations used by NovAtel OEM message fields.");
     init_novatel_oem_enums(enums_mod);
+    init_decoder_tester(m);
```

Comment:
I would move the internal helper classes to a separate submodule as well.

Reply:
I was thinking something like 
```suggestion
    nb::module_ internal_mod = m.def_submodule("internal", "Internal helper functions.");
    init_decoder_tester(internal_mod);
```

Reply:
Ah I see, good suggestion!